### DISCUSSION TITLE
16.1.6

### DISCUSSION THREAD

**Discussion by carloscabanero** [blinksh/blink#1723] [Announcements]
Blink Build is now open to everyone and we are making an important update to our offerings:
- Introducing the **Blink+Build** plan, the most complete dev toolbox on iOS with Shell, Build and Code. To make it even better, we are offering a 1-week free trial, and $9.99/month after that.
- We are **phasing out our Free version in favor of the new Blink+Build trial**. We learnt the hard way that the metered paywall was perceived as an obstacle to the overall experience. We believe the new Blink+Build plan is a more honest offering, allowing everyone to try all you can do in Blink, and after the trial by providing all the value we can as a paid user.
- If you’re currently using the Free plan and want to keep using Blink every day, we wanted to do something special. We are doing our first ever **Blink+ promo for just $14.99 the first year, $19.99 thereafter**. If not, you still can use the Free plan until March 31st to decide, with no metered paywalls.
- Current **Blink+** users, keep on rockin'! Blink+ will keep you updated with all the tools you need for remote dev work, and if you ever need a remote dev environment on demand, you can use Blink Build as an add-on.
- **Blink Classic** (grandfathered version), should continue working all the same, with all the features from v14 and more.
- Please note we still continue to offer our **Community and Open Source** versions for those who want to contribute in a different way.

Other updates include:
- Found root cause for Files.app blocking in some cases at the root container or after a few downloads. Kudos to rrgeorge for the extra help.
- Fixes for our Files.app extension, that could provoke hangs in the connection when keeping its state alive longer than the extension itself.
- Fixed bug on right click contextual menus in the terminal (Issue #1593). And we made the menu cooler too, so you can control parts of the terminal from it instead of doing three finger taps.
- Multiple fixes from TestFlight reports. Thanks to our Community testers!

If you have any comments or suggestions, as always please leave them here!

**Comment by srl295** [comment]
I'm using v16.1.6.723, it says 'Free plan' - came from a v14 owned version.  I haven't gotten around to trying Blink+ or Blink+build, I just use ssh/mosh and sometimes code with my own server.  Does that continue to work? 

I guess the part I'm not sure about is if  'using the free plan' refers to the 'build' features or to the blink app itself or both.

**Reply by carloscabanero** [reply]
Yep, the Blink Classic / Free plan is basically the same functionality as v14. It will just continue being updated under the new app.

**Reply by srl295** [reply]
By the way the top level  appstore link in the README.md in the repo goes to the classic app.  The new appstore link says 'Free, Classic plan $0.00'

**Reply by srl295** [reply]
@carloscabanero thanks for Blink and the quick comeback!

**Reply by carloscabanero** [reply]
Thanks for the heads up. Yeah, we have been so heads down that we have barely paid attention to webpage or even GitHub itself. Will fix it right now.

Let me know if you have any other questions!

**Comment by divinetouch** [comment]
Hi,

I have not used blink for awhile and now all I can see is a page of announcements and I don't know how to get blink working again. I know it's a subscription now but don't know how to do all of that either.
